### ISSUE TITLE
Latitude and longitude information on the globe map

### ISSUE THREAD

**Issue by lmar76** [ESA-VirES/WebClient-Framework#143]
Sometimes it can be necessary to filter data on specific latitude/longitude ranges but it is not easily possible to do it on the globe map because there are no references/information. Why don't add parallels and meridians over the globe according to the zoom level? A further improvement could be the visualization of the coordinates while moving the mouse pointer over the globe, and/or while selecting the area of interest.

**Comment by lubojr** [comment]
I believe this issue will get solved once a pull request #304 is merged.

**Comment by pacesm** [comment]
@lubojr FYI, this requirement is currently on hold (VIR-100).
